lib/Models/Catalog/Ows/WebMapServiceCapabilities.ts

```diff
@@ -190,7 +190,10 @@ export default class WebMapServiceCapabilities {
     readonly [name: string]: CapabilitiesLayer;
   };
 
-  private constructor(readonly xml: XMLDocument, readonly json: any) {
+  private constructor(
```

Comment:
I don't appreciate that constructors with readonly parameters are split over multiple lines even when they fit with room to spare in a single line, but this formatting is also consistent with Prettier 3.4.2.